src/ign.hh

```diff
@@ -21,7 +21,8 @@
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" char *ignitionGazeboVersion();
+extern "C" char *gzSimVersion();
+extern "C" char GZ_DEPRECATED(7) *ignitionGazeboVersion();
```

Comment:
No need to tick-tock, it's not installed

Reply:
https://github.com/gazebosim/gz-sim/pull/1520/commits/b969c8bd754a44e9fe0aba50ae54a131ef015966